domains/h3lios.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "NinjaXility",
+           "email": "",
+           "discord": "511106356267319296",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.IBR3ZrvHG0dyQ1OU_ThEq2VbB_gvPhqPJkyPtwwbODCQENB-T2fstb7JrqYXG9lNaBWBrkeUzrheB5HY5tgEmPDAe6eDnN1D0dk0HKwxySEO7I1wbJsoJUnrWXigmuIYYkCPqHO9bBBD3seh4Y-REvCFI_xjNxBOIkqYiISDG4SJZW_AMoM-4ZwWJOS30ASQA3IM1DrvS3cNeamN0vmy2W16z7PUQxfn3UpwhdalfbzZKgiPowTVnbMT8OZ0IL45VZCmqeikraFf7fUv0PPCk54c07A8mIpYepzyy4hUftsF-ILHDLU_iOMZDHPrMDpPeyFXgYjA1iv1AfgFj6lshA.CFjZ-C3qr_js2SqYJQruNQ.j12mMFCkJhwZpCJXE_JDaY1XgeCUkSwhowfAvMEFrxvTdeqGbeEaHiXN4-oXfM0FuPLNgfsJDTwmva-ekWTAeOtX4a3IBlJ0uuj65_tdMOw.NDTU5ISRijrUIUoDvdYXJw"
+        },
+    
+        "record": {
+            "CNAME": "https://ninjaxility.github.io/"
```

Comment:
CNAME must be a hostname with no protocols or file paths. See docs: https://docs.is-a.dev

Reply:
What Dibster said here. And no, you didn't fix it.